.pre-commit-config.yaml

```diff
@@ -190,8 +190,16 @@ repos:
         language: system
         entry: ./scripts/ci/ci_pylint_tests.sh
         files: ^tests/.*\.py$
+      - id: isort
+        name: Run isort to sort imports
+        language: python
+        entry: isort -rc .
```

Comment:
Remove `-rc .` please. Pre-commit by default passes changed filenames as command arguments (default configuration is pass_filenames: true).

This has a number of advantages:
- in regular pre-commit it will only pass the names of files that changed/were added in this commit/series of commit when pushing (this makes it much faster for smaller commits)
- when you have big number of changes it will automatically split the command to several command run in parallel (equal to the number of threads you have available on your PC) - distributing the changed files between those. If you have 4 CPUs with 2 threads each it will split the big list of files between 8 isort commands and run them in parallel. This means that if you remove all your changes now and run `pre-commit run --all-files` it will run 8 times faster than `isort -rc .`.

Reply:
roger that 👍 I was under the impression that happens only after u set the `files` but obviously I should go read doc first :P